src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -210,17 +211,23 @@ class PaymentMethodList extends Component {
         }
         if (item.type === BUTTON) {
             return (
-                <Button
-                    text={item.text}
-                    icon={item.icon}
-                    onPress={item.onPress}
-                    isDisabled={item.isDisabled}
-                    style={item.style}
-                    iconStyles={item.iconStyles}
-                    success={item.success}
-                    shouldShowRightIcon={item.shouldShowRightIcon}
-                    extraLarge
-                />
+                <FormAlertWrapper
+                    isAlertVisible={false}
```

Comment:
Is this hard-coding of `false` a mistake? When would it ever be `true`?

Reply:
We don't show alerts here for this button. Should I take off the `isRequired` for that param?

Reply:
Yeah, in that case, remove `isRequired` and just have it default to `false` 👍